webextensions/api/browsingData.json

```diff
@@ -26,7 +26,7 @@
               "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browsingData/DataTypeSet",
               "support": {
                 "chrome": {
-                  "version_added": "≤61"
+                  "version_added": "30"
```

Comment:
❌ https://github.com/chromium/chromium/commit/c606b46a266fcef9370b02a2db8f08c8bc57fbcf added this interface as `DataTypes` in [26.0.1404.0](https://github.com/chromium/chromium/releases/tag/26.0.1404.0) before 26.0.1410, and https://github.com/chromium/chromium/commit/c1c8bc9f73e6e4ebd84d812808e699bbe2189fa8#diff-30d9b5f98ec16b49b1a0b7c575d837d5c1a53bc27c057b67a348965aba4eea97 in [27.0.1414.0](https://github.com/chromium/chromium/releases/tag/27.0.1414.0) in 27.0.1453 renamed `DataTypeSet` from `DataTypes`.

Given that the interface is not exposed, we can probably state that the interface was supported from Chrome 26, and I don't think it makes sense to record the alternative name.

```suggestion
                  "version_added": "26"
```